include/SFML/Window/Context.hpp

```diff
@@ -30,6 +30,7 @@
 ////////////////////////////////////////////////////////////
 #include <SFML/Window/Export.hpp>
 #include <SFML/Window/GlResource.hpp>
+#include <SFML/System/Vector2.hpp>
```

Comment:
Can we forward declare instead?

Reply:
Is that possible given how `Vector2` is a class template and `Vector2u` a type alias? I just tried this locally and couldn't get it to work.

Reply:
Yeah, you can't since it's a template